docs/articles/baseplatform/booting-qemu-image.rst

```diff
@@ -10,15 +10,35 @@ virtualization feature if found on the host system and run the virtual
 machine in better performance.
 See KVM feature for QEMU [#qemu_kvm_feature]_.
 
-Installing KVM and its dependencies
+QEMU launch script
 -----------------------------------
 
-On Debian-based systems, the following command can be used to install
-the emulator tool and its dependencies:
+There is a QEMU launch script called ``qemu_launcher``, which is shipped with SDK and also available from yocto builds.
 
 .. code-block:: bash
 
-    $ sudo apt-get install qemu-kvm libvirt-clients libvirt-daemon-system
+	$ ./qemu_launcher -h
+	usage: scripts/qemu_launcher [options]
+	[options] is any of the following:
+	  -f | --folder           	Specify a source folder of the QEMU image
+								Defaults to tmp/deploy/images/qemux86-64
+	  -i | --hda              	Specify rootfs file. Defaults to core-image-ntg-x-qemux86-64.ext4
+	  -p | --ssh_port         	SSH port to connet to QEMU. Defaults to 1234
+	  -n | --smp              	Number of cpu cores. Defaults to 4
+	  -k | --kernel_path      	By default, kernel_path is the same as folder, but can be overwritten by using this option
+	  -c | --cmdline          	Specify kernel command line. This would allow power users to change vga or root, etc.
+								Defaults to "vga=0 uvesafb.mode_option=1280×720-32 root=/dev/hda console=ttyS0 rw oprofile.timer=1"
+	  -m | --mem              	Maximum amount of guest memory (Size is in megabytes)
+								Defaults to 512
+	  -t | --gdb_port         	The port is forwarded so that gdbserver can listen on it
+								Then gdb can be run from develop machine to connect to the port for remote debugging
+								Defaults to 3333
+	  -d | --dev              	Run QEMU development image instead of normal image
+								Defaults to core-image-ntg-x-dev-qemux86-64.ext4
+	  -s | --static           	Run QEMU development image with static IP (192.168.7.2)
```

Comment:
Is 192.168.7.2 the default or is it hard-coded? I don't like hard-coded values.

Reply:
I think it is better to involve Basem or Jonathan for this discussion, since they know better than me. I remember we have this static IP in our local.conf, but maybe it is needed to upstream to meta-pelux as well.

Reply:
ping @BassemMohsen @jonte

Reply:
Looks hardcoded. The project where this script stems from uses this IP (always). I think it would make sense to make the IP configurable, so that this is usable by other projects as well.

Reply:
So the runqemu script relies on the default variables of:
TEST_TARGET_IP = "192.168.7.2"
TEST_SERVER_IP = "192.168.7.1"

Booting the qemu outside testing environment, could be launched to any values.
The instruction for booting with static ip address however give you opportunities to use TAP/TUN to get internet access to the qemu machine. and it doesn't matter changing the default values as long as user know what he's doing like updating default value here:
/etc/systemd/network/10-pelux-ethernet.network

Reply:
The environment variables lives here:
meta/lib/oeqa/runexported.py